test/parallel/test-cluster-debug-port.js

```diff
@@ -1,42 +1,69 @@
 'use strict';
+
+// this is hack to make master process think it is running with --debug
+if (process.execArgv.length == 0) {
+  process.execArgv = ['--debug'];
+}
+
 require('../common');
 const assert = require('assert');
 const cluster = require('cluster');
 
 if (cluster.isMaster) {
-
   function checkExitCode(code, signal) {
     assert.strictEqual(code, 0);
     assert.strictEqual(signal, null);
   }
 
-  console.log('forked worker should not have --debug-port');
-  cluster.fork().on('exit', checkExitCode);
+  console.log('if master is debugging with default port, ' +
+    'forked worker should have debugPort, with offset = 1');
+  cluster.fork({
+    portSet: process.debugPort + 1
+  }).on('exit', checkExitCode);
```

Comment:
Whoops! Ignore me! It won't work reliably on an `exit` handler on `process` but should be fine on child_process (I think).